near-token-contract/src/lib.rs

```diff
@@ -74,7 +74,7 @@ impl Contract {
     /// [`Role::MetaDataUpdater`] to it.
     #[init]
     #[payable]
-    pub fn new() -> Self {
+    pub fn new(admin: AccountId) -> Self {
```

Comment:
```suggestion
    pub fn new(super_admin: AccountId) -> Self {
```
To avoid confusion since Acl has regular admins (for single roles) and super admins.

The function's doc comment should then be updated to replace `factory` in the last paragraph by `[super_]admin`